addons/OPT/dynmarkerverwaltung/fn_clientInit.sqf

```diff
@@ -70,6 +70,37 @@ GVAR(dialogCheck) =
 	
 ] call CFUNC(addPerFrameHandler);
 
+//Verarbeitung EIN
+DFUNC(VerarbeitungEin) = 
```

Comment:
Auch hierfür sollte eine eigene Datei erstellt werden und die Funktion in die `modules.hpp` eingetragen werden